src/xasl/xasl_iteration.hpp

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#ifndef _XASL_ITERATION_HPP_
+#define _XASL_ITERATION_HPP_
+
+#include "xasl.h"
+#include <functional>
+
+namespace cubxasl
+{
+  template<typename T>
+  using XASL_iteration_function = std::function<T (XASL_NODE *)>;
```

Comment:
```suggestion
  using xasl_iteration_function = std::function<T (XASL_NODE *)>;
```
여기서도 소문자 이름 규칙을 지켜야 할 것 같습니다.

Reply:
https://github.com/CUBRID/cubrid/pull/6442/commits/5d04681f3d9c49f77f8ea9a806e681a6caa1f29f 에서 적용하였습니다.

Reply:
확인했습니다. resolve 하겠습니다.